Python/Github_Traffic/README.md

```diff
@@ -0,0 +1,85 @@
+# Github_Traffic
+
+This script, can help one determine all the unique viewers on his/her Repository.
+By using `access-token` one can determine the viewers and number of time they viewed the Repo.
+Currently data of only 14 days is possible (limitation of GitHib-API), but in future by using the GraphQL queries, this limitation can be answered.
+Te script uses PickleDB, the data can be imported in CSV as well, although the functionality hasn't been added till this point of time.
+
+## Note
+
+This script can be converted in a Bot, Action or GitHub app!!!
```

Comment:
Some of the Scripts have a lot of potential into being converted into full-fledged app. I am in works of shifting this Project under an Organization where contributors and maintainers can be better managed. Thanks for the suggestion 👍

Reply:
Seems really interesting :)

Reply:
@HarshCasper Can you also check the `discussion` once, I made a similar suggstion/request